### PR TITLE
Removed Powermock from JS based adaptive script tests.

### PR DESCRIPTION
**Pull request by ruwanta**
### Proposed changes in this pull request

JS Adaptive authentication script tests contains too many Powermock code. This causes few issues.
1. Some JDK s do not like powermock
2. Bytecode instrumentation causes class conflicts
3. Can not run independent tests in Intellij Idea

This contribution removes powermock from these tests so that it is easy to test and follow TDD.

-

### When should this PR be merged

SAPA

### Follow up actions

-

#### General

- [ ] **Is this PR explained thoroughly?** All code changes must be accounted for in the PR description.
- [ ] **Is the PR labeled correctly?**

#### Functionality

- [ ] **Are all requirements met?** Compare implemented functionality with the requirements specification.
- [ ] **Does the UI work as expected?** There should be no Javascript errors in the console; all resources should load. There should be no unexpected errors. Deliberately try to break the feature to find out if there are corner cases that are not handled.

#### Code

- [ ] **Do you fully understand the introduced changes to the code?** If not ask for clarification, it might uncover ways to solve a problem in a more elegant and efficient way.
- [ ] **Does the PR introduce any inefficient database requests?** Use the debug server to check for duplicate requests.
- [ ] **Are all necessary strings marked for translation?** All strings that are exposed to users via the UI must be [marked for translation](https://docs.djangoproject.com/en/1.10/topics/i18n/translation/).

#### Tests

- [ ] **Are there sufficient test cases?** Ensure that all components are tested individually; models, forms, and serializers should be tested in isolation even if a test for a view covers these components.
- [ ] **If this is a bug fix, are tests for the issue in place?**  There must be a test case for the bug to ensure the issue won’t regress. Make sure that the tests break without the new code to fix the issue.
- [ ] **If this is a new feature or a significant change to an existing feature?** has the manual testing spreadsheet been updated with instructions for manual testing?

#### Security

- [ ] **Confirm this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.**
- [ ] **Are all UI and API inputs run through forms or serializers?**
- [ ] **Are all external inputs validated and sanitized appropriately?**
- [ ] **Does all branching logic have a default case?**
- [ ] **Does this solution handle outliers and edge cases gracefully?**
- [ ] **Are all external communications secured and restricted to SSL?**

#### Documentation

- [ ] **Are changes to the UI documented in the platform docs?** If this PR introduces new platform site functionality or changes existing ones, the changes should be documented.
- [ ] **Are changes to the API documented in the API docs?** If this PR introduces new API functionality or changes existing ones, the changes must be documented.
- [ ] **Are reusable components documented?** If this PR introduces components that are relevant to other developers (for instance a mixin for a view or a generic form) they should be documented in the Wiki.

### CODE REVIEW

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandlerClaimsTest.java [hunk 1]
```diff
@@ -89,15 +63,12 @@ public void testHandleClaimHandling() throws Exception {
...
-        FrameworkServiceDataHolder.getInstance().setRealmService(mockRealmService);
-        PowerMockito.when(mockUserStoreManager.getUserClaimValues(anyString(),
-                eq(new String[]{"http://wso2.org/claims/lastname"}),
-                anyString())).thenReturn(Collections.singletonMap("http://wso2.org/claims/lastname", "lastNameValue"));
+//
+//        FrameworkServiceDataHolder.getInstance().setRealmService(mockRealmService);
+//        PowerMockito.when(mockUserStoreManager.getUserClaimValues(anyString(),
+//                eq(new String[]{"http://wso2.org/claims/lastname"}),
+//                anyString())).thenReturn(Collections.singletonMap("http://wso2.org/claims/lastname",
+//                "lastNameValue"));
```
Comment (senthalan) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandlerClaimsTest.java:71]: Shall we remove these commented lines

File: test-utils/org.wso2.carbon.identity.testutil/src/main/java/org/wso2/carbon/identity/common/testng/realm/UserStoreModel.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2021. WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (janakamarasena) [inline test-utils/org.wso2.carbon.identity.testutil/src/main/java/org/wso2/carbon/identity/common/testng/realm/UserStoreModel.java:2]:
```suggestion
 * Copyright (c) 2021. WSO2 Inc. (http://www.wso2.org).
```

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandlerClaimsTest.java [hunk 3]
```diff
@@ -89,15 +63,12 @@ public void testHandleClaimHandling() throws Exception {
...
-        RealmService mockRealmService = mock(RealmService.class);
-        UserRealm mockUserRealm = mock(UserRealm.class);
-        AbstractUserStoreManager mockUserStoreManager = PowerMockito.mock(AbstractUserStoreManager.class);
-        when(mockRealmService.getTenantUserRealm(anyInt())).thenReturn(mockUserRealm);
-        when(mockUserRealm.getUserStoreManager()).thenReturn(mockUserStoreManager);
-        FrameworkServiceDataHolder.getInstance().setRealmService(mockRealmService);
-        PowerMockito.when(mockUserStoreManager.getUserClaimValues(anyString(),
-                eq(new String[]{"http://wso2.org/claims/lastname"}),
-                anyString())).thenReturn(Collections.singletonMap("http://wso2.org/claims/lastname", "lastNameValue"));
+//
```
Comment (madurangasiriwardena) [inline components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandlerClaimsTest.java:66]: Let's remove the commented code

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandlerClaimsTest.java (hunk 1 above)
Comment (ruwanta) [reply components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandlerClaimsTest.java]: +1, done

File: components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandlerClaimsTest.java (hunk 3 above)
Comment (ruwanta) [reply components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/handler/sequence/impl/GraphBasedSequenceHandlerClaimsTest.java]: +1, done

**Review by senthalan — APPROVED** [review]

**Review by janakamarasena — APPROVED** [review]

**Pull request merged**